src/apps/backend/modules/account/account-service.ts

```diff
@@ -1,5 +1,5 @@
-import { OtpService } from '../otp';
-import { PasswordResetTokenService } from '../password-reset-token';
+import { OtpService } from '../authentication/otp';
```

Comment:
Here, and other places, we should get rid relative paths. We should expose via `./authentication`.

OTPService should be proxied via AuthenticationService and not directly exposed.